libs/design/src/molecules/paginator/paginator.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../scss/daff-util';
+
+:host {
+	display: flex;
+	justify-content: center;
+
+	.daff-paginator {
```

Comment:
`.daff-paginator` doesn't need to be inside of `:host`?

Reply:
moved